docs/ml-features.md

```diff
@@ -183,6 +183,75 @@ for words_label in wordsDataFrame.select("words", "label").take(3):
 </div>
 </div>
 
+## OneHotEncoder
+
+One-hot encoding is a way of formatting categorical features as input into machine learning algorithms. It maps a column of label indices to a column of binary vectors, with at most a single one-value. The [OneHotEncoder](api/scala/index.html#org.apache.spark.ml.feature.OneHotEncoder) class provides this functionality. By default, the resulting binary vector has a component for each category, so with 5 categories, an input value of 2.0 would map to an output vector of (0.0, 0.0, 1.0, 0.0, 0.0). If the includeFirst is set to false, the first category is omitted, so the output vector for the previous example would be (0.0, 1.0, 0.0, 0.0) and an input value of 0.0 would map to a vector of all zeros. Including the first category makes the vector columns linearly dependent because they sum up to one.
```

Comment:
Can you please add something like, "This encoding allows algorithms which expect continuous features to use categorical features as well; for example Logistic Regression requires continuous features, but it can use categorical features after one-hot encoding."

Could you please add a Wikipedia link? [http://en.wikipedia.org/wiki/One-hot]

"the includeFirst" --> "the `includeFirst` parameter"